nilearn/plotting/img_plotting.py

```diff
@@ -36,6 +36,44 @@
 ################################################################################
 # Core, usage-agnostic functions
 
+def _get_plot_stat_map_params(stat_map_img, vmax, symmetric_cbar, kwargs, force_min_stat_map_value=None):
+# make sure that the color range is symmetrical
```

Comment:
Maybe a small docstring to at least say that is some utility function for common colorbar-related functionality b/w plot_stat_map and plot_glass_brain.